src/bomberman.hpp

```diff
@@ -14,9 +14,13 @@
 #include <glm/gtc/matrix_transform.hpp>
 #include <glm/ext.hpp>
 #include <entt/entt.hpp>
+//#include "../lib/irrlkang/include/irrKlang.h"
+#include <irrKlang.h>
 
 #define ASSET_PATH "assets/"
 
+using namespace irrklang;
```

Comment:
We can't use using namespace (PDF rules :( ), so instead you can use `namespace i = irrklang`